### PR TITLE
Update Set-CsOnlineVoicemailPolicy.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updating with new dual language parameters

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOnlineVoicemailPolicy.md [hunk 1]
```diff
@@ -151,6 +160,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PrimarySystemPromptLanguage
+The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineVoicemailPolicy.md:164]:
```suggestion
The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

File: skype/skype-ps/skype/Set-CsOnlineVoicemailPolicy.md [hunk 2]
```diff
@@ -151,6 +160,38 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SecondarySystemPromptLanguage
+The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineVoicemailPolicy.md:180]:
```suggestion
The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

**Review by get-itips — APPROVED** [review]
I am ok once @dariomws suggestions are applied, and also adding the same question as https://github.com/MicrosoftDocs/office-docs-powershell/pull/7804

**Review by tseward — APPROVED** [review]

**Review by dariomws — COMMENTED** [review]
Another suggestion. You can use a short link to files in the same location.

**Comment by yogkumgit** [comment]
@Kateyanne please merge this PR. Thanks

**Pull request merged**